packages/admin-client/src/components/CreateProxyMaps.tsx

```diff
@@ -0,0 +1,70 @@
+import { Button, Dialog, DialogActions, DialogContent, DialogTitle, FormControl, FormHelperText, Input, InputLabel } from '@mui/material'
+import axios from 'axios';
+
+interface Props {
+  open: boolean;
+  setOpen: (open: boolean) => void;
+}
+
+export default function CreateProxyMaps({ open, setOpen }: Props) {
+
+  const handleClose = () => {
+    setOpen(false);
+  };
+
+  const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    const form = e.currentTarget;
+    const model = form.querySelector('#model-name') as HTMLInputElement;
+    const baseUrl = form.querySelector('#base-url') as HTMLInputElement;
+    const key = form.querySelector('#key') as HTMLInputElement;
+
+    axios.post(`${import.meta.env.VITE_BACKEND_BASE_URL}/deepchat-proxy`, {
```

Comment:
NOTE: In the future we will want to replace the manual calls with generated client code